js/src/modal.js

```diff
@@ -434,6 +433,13 @@ EventHandler.on(document, EVENT_CLICK_DATA_API, SELECTOR_DATA_TOGGLE, function (
   data.toggle(this)
 })
 
+EventHandler.on(document, EVENT_CLICK_DATA_API, SELECTOR_DATA_DISMISS, function (event) {
+  const target = getElementFromSelector(this) || this.closest(SELECTOR)
+  const modal = Modal.getInstance(target) || new Modal(target)
```

Comment:
Why not `getOrCreateInstance`? 😁

Reply:
hahahah, cause the pr is older than `getOrCreateInstance`

I 'll update it ;)